README.md

```diff
@@ -121,6 +121,13 @@ rails server
 - Twitter: [@rbreva75](https://twitter.com/rbreva75)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/r-mauricio-mendoza-huerta/)
 
+👤 **Didier Ganthier**
+
+- GitHub: [@didierganthier](https://github.com/didierganthier)
+- Twitter: [@didierganthier_](https://twitter.com/didierganthier_)
+- LinkedIn: [Didier Peran Ganthier](https://linkedin.com/in/didierganthier)
+
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🔭 Future Features <a name="future-features"></a>
```

Comment:
- As pointed out by the previous reviewer, your license file still contains obsolete information as you can see in the screenshot below:

![Capture](https://user-images.githubusercontent.com/63087491/215939745-bc07192b-e033-4e72-be52-0009092d8c5e.PNG)

- You really need to update the required information(date and name)